### PR TITLE
Registering TEAM The Bug Slayer

### PR DESCRIPTION
**Pull request by Amik-Sen**
# Application for Registering Team to CODE CAMP 1.0

This pull request template helps you complete an application to the **CODE CAMP 1.0 HACKATHON**. Use the checklist below to verify that you have followed the instructions correctly. 

**(Put x inside [] like [x])**

## Select the type of Pull Request

[x] Registration Pull request.

[] Submission Pull request.

## Checklist ✅  

- [x] I have read the instructions on the README file before submitting my application and gone through [CODE_OF_CONDUCT](). 
- [x] I have Submitted my team's details by making a folder of my team as instructed in [How to send Pull Request page](https://codecamphackathon.netlify.app/makingRegistration/).
- [x] I have used the Markdown file template to add my information for the Hackathon.
- [x] I have given all the necessary details as mentioned in template index.md file and the details are correct and best to my knowledge.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the Details added to the template will be used as a means of communication at the time of result declaration.
- [x] I have added the [event](https://calendar.google.com/calendar/r/eventedit/copy/MHUzM2pvaDQwM3R1cDdxMmdjMmQyYjZpaHAgN3NhMzI0N2FpMzkzczAzbWo0YWMxNWVzdG9AZw) to my calendar.

### CODE REVIEW

File: content/teams/The Bug Slayers/index.md [hunk 1]
```diff
@@ -1,55 +1,68 @@
...
-_**Application-Type**_ :   Web Application
+_**Application-Type**_ :   Web Application 
+
+_**Idea-Description**_ :   
+
+_**Tech-Stack-Used**_ :   
+
+_**GitHub-Link**_ :   
+
+_**Hosted-Link**_ :
```
Comment (Uyadav207) [inline content/teams/The Bug Slayers/index.md:29]: can you Mention more details to these. Also leave Hosted Link : write N/A for now

File: content/teams/The Bug Slayers/index.md [hunk 2]
```diff
@@ -1,55 +1,68 @@
...
+ 
+_**Technical-Skills**_ : BootStrap, Firebase, HTML, CSS, Javascript, C++
 
-_**Technical-Skills**_ : Vue.js,HTML,CSS,JS,Firebase,C,C++,Java,Python,DSA,Github,Virtual Prototyping,SQL
-
-_**Specialization**_ : Vuejs
+_**Specialization**_ : Front-End Developer
 
-_**GitHub-ID**_ :  49827700
+_**GitHub-ID**_ :  NancyAgrawal
```
Comment (Uyadav207) [inline content/teams/The Bug Slayers/index.md:50]: complete link add https://github.com/

**Review by Uyadav207 — CHANGES_REQUESTED** [review]

**Comment by Uyadav207** [comment]
@Amik-Sen , Kindly do the changes ASAP

**Comment by Saikumarksk** [comment]
can anyone confirm our team's registration

**Comment by Uyadav207** [comment]
> can anyone confirm our team's registration

please do not spam other's PR. Will review your PR. Please reply to my messages in your PR sessions. I have requested you for some changes. Please Do that ASAP

**Comment by Uyadav207** [comment]
@Amik-Sen , Today is last day of  submission after that will not merge your PR, please have a look

**Comment by Uyadav207** [comment]
# YOU NEED TO CORRECT THE INFORMATION AS INSTRUCTED BY ME ELSE IF NOT REGISTER, YOU WILL BE IN NO POSITION TO PARTICIPATE IN THIS HACKATHON. PLS REGISTER BEFORE 10:00 PM TODAY ELSE WILL NOT MERGE YOUR PROFILE. @Amik-Sen

**Pull request closed without merging**